templates/default/mongod.service.erb

```diff
@@ -12,6 +12,8 @@ PIDFile=/var/run/mongod/mongod.pid
 ExecStart=/usr/bin/mongod -f /etc/mongod.conf --pidfilepath /var/run/mongod/mongod.pid --fork
 TimeoutStopSec=5
 KillMode=mixed
+LimitNOFILE=64000
+LimitNPROC=64000
```

Comment:
64000 processes seems like a lot?

Reply:
It is actually threads on linux, and it is a lot but that is what the docs suggest and far be it from me to argue.

Reply:
🤷